src/components/cuda/cupti_common.c

```diff
@@ -248,6 +258,10 @@ static int util_load_cuda_sym(void)
     papi_errno = load_cuda_sym();
     papi_errno += load_cudart_sym();
     papi_errno += load_cupti_common_sym();
+    if (linked_cudart_path) {
+        papi_free((void*) linked_cudart_path);
```

Comment:
Do you need to cast to void *?

Reply:
`linked_cudart_path` is a `const char *`. So freeing it raises a warning `ignores const qualifier`. Casting it to `void *` resolves the warning.

Reply:
This was also a bug. It shouldn't be freed in `util_load_cuda_sym`. Now it is freed in `cupti_shutdown`